Reneiski_Homework1/ReneiskiSelenium11/Program.cs

```diff
@@ -0,0 +1,239 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace Reneiski_Homework1
+{
+    public class Selenium
+    {
+        private IWebDriver _driver;        
+        private WebDriverWait _driverWait;
+        private Actions _driverActions;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+            _driverWait = new WebDriverWait(_driver, TimeSpan.FromSeconds(1000));
+            _driverActions = new Actions(_driver);
+        }
+
+        [Test]
+        public void FirstCheckBoxTest()
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/checkbox");
+
+            var firstCheckBox = _driver.FindElement(By.XPath("//*[@class='rct-icon rct-icon-uncheck']"));
+            firstCheckBox.Click();
+
+            var checkedCheckbox = _driver.FindElement(By.XPath("//*[@class='rct-icon rct-icon-check']"));
+            var isCheckboxChecked = checkedCheckbox.GetAttribute("class").Contains("check");
+            
```

Comment:
you may remove an empty line here
empty line should help to split code to kind of a logical blocks, in your case you have checked whether checkbox is checked, then goes Assert and it's a logically connected lines. Please check the whole file and fix

Reply:
Fixed